test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+mvn fmt:format
```

Comment:
@AndrienkoAleksandr looks like you forced pushed smth. unexpected ;-)

Reply:
Thanks, I removed. Sorry I did force push because I re-based code to the latest master. Maybe it was a bad idea.